go.mod

```diff
@@ -21,13 +21,20 @@ require (
 )
 
 require (
+	cel.dev/expr v0.18.0 // indirect
```

Comment:
Any idea why we add a bunch of indirect dependencies without changes to the direct dependencies?

Reply:
This indirect package is not added explicitly, Seems like this package is downloaded by some other direct dependency